benchmarks/distributed/rpc/ps/launcher.py

```diff
@@ -0,0 +1,372 @@
+                configurations.trainer_count,
+                configurations.parameter_server_count
+            )
+            rpc_backend_options.set_device_map(
+                ps_name,
+                {rank: ps_rank}
+            )
+        trainer_name = get_name(
+            rank,
+            configurations.trainer_count,
+            configurations.parameter_server_count
+        )
+        rpc.init_rpc(
+            trainer_name,
+            rank=rank,
+            world_size=world_size,
+            rpc_backend_options=rpc_backend_options
+        )
+    rpc.shutdown()
+
+
+def load_configurations(args):
+    benchmark_config_file = "configurations/benchmark_configurations.json"
+    benchmark_config = json.load(
+        open(
+            os.path.join(Path(__file__).parent, benchmark_config_file),
+            "r"
+        )
+    )[args.bconfig_id]
+    configurations = BenchmarkConfigurations(**benchmark_config)
+    return configurations
+
+
+def get_data(data_class, data_config):
+    data_class = get_benchmark_data_map()[data_class]
+    return data_class(**data_config)
+
+
+def load_data(args):
+    data_config_file = "configurations/data_configurations.json"
+    data_config = json.load(
+        open(
+            os.path.join(
+                Path(__file__).parent, data_config_file
+            ),
+            "r"
+        )
+    )[args.dconfig_id]
+    return get_data(data_config["data_class"], data_config["configurations"])
+
+
+def get_model(model_class, model_config):
+    model_class = get_benchmark_model_map()[model_class]
+    return model_class(**model_config)
+
+
+def load_model(args):
+    model_config_file = "configurations/model_configurations.json"
+    model_config = json.load(
+        open(
+            os.path.join(
+                Path(__file__).parent, model_config_file
+            ),
+            "r"
+        )
+    )[args.mconfig_id]
+    return get_model(model_config["model_class"], model_config["configurations"])
+
+
+def load_trainer_configurations(args):
+    trainer_config_file = "configurations/trainer_configurations.json"
+    trainer_config = json.load(
+        open(
+            os.path.join(
+                Path(__file__).parent, trainer_config_file
+            ),
+            "r"
+        )
+    )[args.tconfig_id]
+    return trainer_config
+
+
+def load_parameter_server_configurations(args):
+    if args.pconfig_id == "None":
+        return None
+    parameter_server_config_file = "configurations/parameter_server_configurations.json"
+    parameter_server_config = json.load(
+        open(
+            os.path.join(
+                Path(__file__).parent, parameter_server_config_file
+            ),
+            "r"
+        )
+    )[args.pconfig_id]
+    return parameter_server_config
+
+
+def main():
+    parser = argparse.ArgumentParser(description="RPC PS Benchmark")
+    parser.add_argument(
+        "--bconfig_id",
+        type=str,
+        default="1",
```

Comment:
you can add the `help` parameter to provide information about each argument

Reply:
okay.